portfolio/src/main/webapp/index.html

```diff
@@ -17,6 +17,9 @@ <h1>Anthony's Portfolio</h1>
       <p>Click here to get a random message:</p>
       <button onclick="addRandomGreeting()">Hello</button>
       <div id="greeting-container"></div>
+
+      <button onclick="showtime()">Click here</button>
```

Comment:
The function you've defined is actually called showServerTime(), make sure to use that here